doc/src/reference/py.rst

```diff
@@ -146,6 +146,13 @@ module
     :param list[module] mod_args: optional list of module arguments to the operator.
     :rtype instruction
 
+.. py:method:: add_literal(data)
+
+    Adds constant or literal data of provided shape into the module from python array or numpy array.    
```

Comment:
This should be: `Adds constant or literal data of provided shape into the module from python buffer which includes numpy arrays`.

Reply:
Done